index.json

```diff
@@ -239,10 +239,10 @@
   ],
   "fontFamily": {
     "arialBlack": "'Arial Black', 'Arial Bold', Gadget, sans-serif",
-    "lato": "Lato, -apple-system, 'Segoe UI', Helvetica, Arial, sans-serif",
-    "mono": "'Roboto Mono', Monaco, 'Lucida Console', 'Courier New', Courier, monospace",
-    "monoSystem": "Monaco, 'Lucida Console', 'Courier New', Courier, monospace",
-    "sansSystem": "-apple-system, 'Segoe UI', Helvetica, Arial, sans-serif",
+    "lato": "Lato, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'",
+    "mono": "'Roboto Mono', ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace",
+    "monoSystem": "ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace",
```

Comment:
This looks like it matches GitHub:
![image](https://user-images.githubusercontent.com/806152/210859869-98e79ab0-d2ff-45b1-af84-bd15c99044d0.png)